applications/vanilla/models/class.categorymodel.php

```diff
@@ -40,6 +40,8 @@ class CategoryModel extends Gdn_Model {
     /** Flag for aggregating discussion counts. */
     const AGGREGATE_DISCUSSION = 'discussion';
 
+    const DISPLAY_FLAT = 'Flat';
```

Comment:
Since this constant was introduced, you should introduce the other constants from the display as and see where they might be referenced within this class.